gallery-so/gallery
# replace shop link with faq and update mobile faq too

### Summary of Changes

replace shop link with faq in web left hand nav, and update mobile faq icon + faq + url

also updates SidebarIcon to support to or href

### Demo or Before/After Pics
Sidebar:
<img width="166" alt="CleanShot 2023-11-15 at 17 28 12@2x" src="https://github.com/gallery-so/gallery/assets/80802871/f7185183-067d-4fbe-92dd-fbcea6086937">


Mobile: (new icon and url)
![CleanShot 2023-11-15 at 17 44 32](https://github.com/gallery-so/gallery/assets/80802871/6844969b-8d9e-49e2-aee1-0e58db9ae8b6)
![CleanShot 2023-11-15 at 17 43 07](https://github.com/gallery-so/gallery/assets/80802871/e7b3497c-0d7d-49b2-a2f8-436755b77dae)


Regression check: existing question mark icon in Editor looks correct
<img width="178" alt="CleanShot 2023-11-15 at 17 28 58@2x" src="https://github.com/gallery-so/gallery/assets/80802871/5a47f400-4441-4e61-b3d9-f11a9685bde8">


### Edge Cases
na

### Testing Steps
on web:
check the bottom of the left hand nav. should link to faq in notion


on mobile:
open settings, confirm FAQ link works

### Checklist

Please make sure to review and check all of the following:
